python/runKoaTrack.py

```diff
@@ -72,6 +72,8 @@
     not check_stage_success(pathname + f"/Koala_Track_{start_evt}.root")
     or force_level == 1
 ):
+    os.chdir(macropath)
+    os.chdir(macropath)
```

Comment:
why two times?

Reply:
I'm not sure, it might be that it just a problem from copying the previous code